libbeat/kibana/client.go

```diff
@@ -153,7 +155,8 @@ func NewClientWithConfigDefault(config *ClientConfig, defaultPort int) (*Client,
 		headers.Set(k, v)
 	}
 
-	rt, err := config.Transport.Client()
+	userAgent := useragent.UserAgent(beatname)
```

Comment:
what if beatname is empty? we should not try to add any header and risk overriding already valid value with nothing

Reply:
Added a default `Libbeat` value